GUI.py

```diff
@@ -78,6 +89,18 @@ def __init__(self, parent: MyFrame) -> None:
         super(MyPanel, self).__init__(parent=parent)
```

Comment:
It python-2 inheritance-style. I think better use the common way for python 3:

```suggestion
        super().__init__(parent=parent)
```

Fix it everywhere